openrr-apps/src/bin/robot_teleop.rs

```diff
@@ -41,7 +41,22 @@ async fn main() -> Result<(), Error> {
         panic!("No valid nodes");
     }
 
-    let switcher = Arc::new(ControlNodeSwitcher::new(nodes, client.clone()));
+    let initial_node_index = if teleop_config.initial_mode.is_empty() {
+        0
+    } else if let Some(initial_node_index) = nodes
+        .iter()
+        .position(|node| node.mode() == teleop_config.initial_mode)
+    {
+        initial_node_index
+    } else {
+        0
```

Comment:
I think it's better to add warning to show that teleop_config.initial_mode is not found.
Or It might be better to die.